sphinx_needs_enterprise/extensions/extension.py

```diff
@@ -180,10 +180,20 @@ def _extract_data(self, data, options):
                         f'Got {type(selector)} with value "{selector}"'
                     )
                 if isinstance(selector, str):
-                    # Set the "hard-coded" string as value
-                    need_values[name] = selector
+                    # Combine "hard-coded" string and dynamic value and set as value
+                    if re.search(r"{{\w+?}}", selector) is not None and isinstance(
+                        re.search(r"{{\w+?}}", selector), re.Match
+                    ):
+                        need_values[name] = jinja_parse(item, selector)
+                    else:
```

Comment:
I think  you can always set the value of `jinja_parse()`.

If not ``{{...}`` is in the string/selector, jinja will return the untouched string, which you also do in the else-part.

Reply:
To ensure that jinja will return the untouched string if `{{...}}` is not in the string/selector, I made changes to this line:
```python
if isinstance(selector, str):
    # Set the "hard-coded" string or
    # combine the "hard-coded" string and dynamic value
    selector = jinja_parse(item, selector)
    # Set the returned string as value
    need_values[name] = selector

```